src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -125,4 +164,21 @@ export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDest
       });
     }
   }
+
+  private initializeDragAndDrop(): void {
+    this.dragulaService.drag.subscribe(([, source]: Array<HTMLElement>) =>
+      source.classList.add('sky-repeater-item-dragging')
+    );
+
+    this.dragulaService.dragend.subscribe(([, source]: Array<HTMLElement>) =>
```

Comment:
Should we make sure these couple `subscribe`s get destroyed properly?

Reply:
Good catch. Fixed.